automerge-protocol/src/lib.rs

```diff
@@ -12,7 +12,7 @@ pub struct ActorID(pub String);
 
 impl ActorID {
     pub fn random() -> ActorID {
-        ActorID(uuid::Uuid::new_v4().to_string())
+        ActorID(hex::encode(uuid::Uuid::new_v4().to_string()))
```

Comment:
It would probably be better to directly encode the `as_bytes` of the UUID I think. Something like:

```
ActorID(hex::encode(uuid::Uuid::new_v4().as_bytes()))
```

Reply:
Sure thing!

Reply:
⚡ https://github.com/automerge/automerge-rs/pull/18/commits/a28f3990c7874d7e140b680137529f6b201ad6a5